client/server/pages/index.js

```diff
@@ -151,7 +152,10 @@ function getDefaultContext( request, entrypoint = 'entry-main' ) {
 		featuresHelper: !! config.isEnabled( 'dev/features-helper' ),
 		devDocsURL: '/devdocs',
 		store: reduxStore,
-		addEvergreenCheck: target === 'evergreen' && calypsoEnv !== 'development',
+		addEvergreenCheck:
+			request.query.bypassTargetRedirection !== 'true' &&
+			target === 'evergreen' &&
+			calypsoEnv !== 'development',
```

Comment:
What's the purpose of the calypso env check? I noticed that `?forceFallback=true` does not work in dev mode -- it might be helpful for testing the unsupported browser page, not sure if it has an impact otherwise.

Reply:
In dev mode, we don't build the `fallback` target. And we don't even write the compiled files to any directory on disk. Webpack runs inside the Node.js server, compiling and recompiling as the source files change, and serves the JS and CSS assets from memory using `webpack-dev-middleware` plugged into the Express.js server.